src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -9,6 +9,7 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_STUDENT_DISPLAYED_INDEX = "The student index provided is invalid";
     public static final String MESSAGE_STUDENTS_LISTED_OVERVIEW = "%1$d students listed!";
+    public static final String MESSAGE_STUDENTS_FOUND_OVERVIEW = "%1$d students found!";
```

Comment:
Find and Overdue both rely on `MESSAGE_STUDENTS_LISTED_OVERVIEW` for their success message.
Instead of creating a new message, could you just change `MESSAGE_STUDENTS_LISTED_OVERVIEW` instead so both don't end up having to use different messages for the same result?